app/javascript/__tests__/rooms/Rooms.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Rooms from '../../components/rooms/Rooms';
+import RoomsList from '../../components/rooms/RoomsList';
+import RoomsTabs from '../../components/rooms/RoomsTabs';
+import useRooms from '../../hooks/queries/rooms/__mocks__/useRooms';
+
+jest.mock('react-router-dom');
+jest.mock('../../hooks/queries/rooms/useRooms');
+
+const rooms = {
+  data:
+    [
+      {
+        name: 'Home Room',
+        friendly_id: 'dummy_id1',
+      },
+      {
+        name: 'Room 1',
+        friendly_id: 'dummy_id2',
+      },
+      {
+        name: 'Room 2',
+        friendly_id: 'dummy_id3',
+      },
+    ],
+};
+
+describe('Rooms (useRooms query test)', () => {
+  test('Rooms from query match expected rooms', () => {
+    const json = useRooms();
```

Comment:
We are testing a mock here. 
To test queries we should mock the API calls but use the queries original implementation.